### ISSUE TITLE
Same Url is copied to the below newly created fields instead of creating an empty field when "Add another URL" is clicked

### ISSUE THREAD

**Issue by TejaswiKarasani** [OpenRefine/OpenRefine#2381]
**Describe the bug**
When I click "Add another URL" , the new URL box gets the first URL copied in it by default instead of an empty URL field

**To Reproduce**
1. Go to 'Create project'
2. Click on 'Web Addresses URL's'
3.Enter an URL in first URL box and then click on "Add another URL"

**Current Results**
First URL gets copied to another newly created url fields

**Expected behavior**
Empty URL field should be generated

**Screenshots**
![image](https://user-images.githubusercontent.com/46127128/76232092-37b70800-624c-11ea-8648-790e842b7a67.png)

**Desktop (please complete the following information):**
Windows 10
Chrome latest
JDK 23

**OpenRefine (please complete the following information):**
 - Version 3.3

**Comment by TejaswiKarasani** [comment]
@wetneb may I take up the issue?

**Comment by wetneb** [comment]
Sure! It's not completely clear to me if this is a bug or something intended - perhaps the idea was to make it easier to download multiple similar URLs, and so it would be convenient to be able to change the URLs added… It would seem a bit weird though. It might be worth waiting a bit for feedback from others to see if this is a feature or a bug, before you rush to do a fix :)

**Comment by TejaswiKarasani** [comment]
> Sure! It's not completely clear to me if this is a bug or something intended - perhaps the idea was to make it easier to download multiple similar URLs, and so it would be convenient to be able to change the URLs added… It would seem a bit weird though. It might be worth waiting a bit for feedback from others to see if this is a feature or a bug, before you rush to do a fix :)

Yes @wetneb and I would to start working on making my orginal outreachy proposal , now I feel a kind of accustomed to the environment :)

**Comment by TejaswiKarasani** [comment]
> Sure! It's not completely clear to me if this is a bug or something intended - perhaps the idea was to make it easier to download multiple similar URLs, and so it would be convenient to be able to change the URLs added… It would seem a bit weird though. It might be worth waiting a bit for feedback from others to see if this is a feature or a bug, before you rush to do a fix :)

And also I doubt if this made to compensate to not to pop up any error and proceed to download the files when there are empty URL fileds

**Comment by wetneb** [comment]
> I would also like to start working on making my orginal outreachy proposal

I might be wrong but it seems to me that Outreachy does not let interns propose their own projects - I think you would have to apply for the only project we have proposed. If you are also eligible to participate in Google Summer of Code, it is possible to propose your own project there.
If you want to have a chat about that we can probably take this issue to Gitter.

**Comment by TejaswiKarasani** [comment]
> > I would also like to start working on making my orginal outreachy proposal
> 
> I might be wrong but it seems to me that Outreachy does not let interns propose their own projects - I think you would have to apply for the only project we have proposed. If you are also eligible to participate in Google Summer of Code, it is possible to propose your own project there.
> If you want to have a chat about that we can probably take this issue to Gitter.

I meant working on the current proposal itself 😅

**Comment by tfmorris** [comment]
> perhaps the idea was to make it easier to download multiple similar URLs, and so it would be convenient to be able to change the URLs added…

That is my interpretation. I wouldn't blithely change it without getting feedback from users first to make sure that any change is an improvement in their eyes.

**Comment by kushthedude** [comment]
> > perhaps the idea was to make it easier to download multiple similar URLs, and so it would be convenient to be able to change the URLs added…
> 
> That is my interpretation. I wouldn't blithely change it without getting feedback from users first to make sure that any change is an improvement in their eyes.

Agreed with @tfmorris. While importing data from web-URLs mostly users have similar sort of preceding URL, so that user may just change the end part. But feedback from other users may be better.

**Comment by tfmorris** [comment]
One thing which I think would improve the current behavior is that instead of always copying the first URL as it currently does, it copies the most recent URL to the new input field.
